src/libraries/Microsoft.Bcl.Memory/src/PACKAGE.md

```diff
@@ -11,6 +12,7 @@ This library is not necessary nor recommended when targeting versions of .NET th
 
 * Enables the use of `Index` and `Range` types on older .NET platforms.
 * Provides `Base64Url` encoding, decoding, and validation for URL-safe data processing on older .NET platforms.
+* Provides `Utf8` encoding, decoding, and validation for chunked data between UTF-8 and UTF-16 on older .NET platforms.
```

Comment:
I'd remove this here since the minimum supported .NET version by the package has built-in Utf8, we only provide it for netframework/netstandard

Reply:
I made `Utf8` description similar to  `Base64Url`  description. I don't understand what needs to be fixed.

Reply:
`Utf8` is already in net8.0 (minimum .NET platform this package targets) so we only have the type forwarders to the built-in version. This is different from Base64Url which doesn't exist on net8.0 and this package provides it there.

Reply:
I changed the package description, as you recommended.